src/django_components/autodiscovery.py

```diff
@@ -94,4 +100,10 @@ def _import_modules(
         logger.debug(f'Importing module "{module_name}"')
         importlib.import_module(module_name)
         imported_modules.append(module_name)
+
+        # In tests tagged with `@djc_test`, we want to capture the modules that
+        # are imported so we can clean them up between tests.
+        if os.environ.get("DJC_TESTING"):
```

Comment:
So you have to set this env variable when you run with the decorator? I don't see this mentioned in the docs?

Reply:
It's the other way around, the decorator sets this env variable to make django-components start recording which libraries were imported with autodiscovery.

Tbh Not sure if an env var is the right approach. Actually might better to just use a plain module-level variable to keep track of the state. It's an implementation detail, not something that people should rely on. 👀

Reply:
Updated in https://github.com/django-components/django-components/pull/1008/commits/070ba7cb93b89a2613ab8b8fa1466cd99e2d823e